backoffice/src/test/java/com/wedriveu/backoffice/analytics/VehicleCounterGenerator.java

```diff
@@ -0,0 +1,19 @@
+package com.wedriveu.backoffice.analytics;
+
+import com.wedriveu.shared.rabbitmq.message.VehicleCounter;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+public class VehicleCounterGenerator {
```

Comment:
Where is all the PPS course knowledge about design patterns & code? Here i miss javadoc and it seems like a `Factory` pattern but you have to implement it right